notebooks/analyze_looping_across_lending_protocols.ipynb

```diff
@@ -206,21 +211,23 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Tokens to analyze\n",
+    "# Tokens\n",
     "tokens = [\"ETH\", \"wBTC\", \"USDC\", \"DAI\", \"USDT\", \"wstETH\", \"LORDS\", \"STRK\", \"UNO\", \"ZEND\"]\n",
     "\n",
     "# Initialize a dictionary to keep track of borrowed and collateral assets\n",
     "borrowed_assets = {}\n",
     "collateral_assets = {}\n",
+    "all_users = set()\n",
+    "common_users = {}\n",
     "\n",
     "# Extract and split the collateral and debt information for each protocol\n",
-    "for protocol, df in dataframes.items():\n",
+    "for protocol, protocol_df in dataframes.items():\n",
```

Comment:
```suggestion
    "for protocol, protocol_loan_states in loan_states.items():\n",
```
Avoid using names like `df` which are not very descriptive.

Reply:
Done! ✅